udata_front/theme/gouvfr/assets/js/pages/Admin/Members/Members.vue

```diff
@@ -45,49 +45,57 @@
+    <h2
+      class="subtitle subtitle--uppercase fr-mb-0"
+      :class="{'fr-mt-n3v': membershipRequests.length, 'fr-mt-5v': !membershipRequests.length}"
+    >
       {{ t("{n} members", {n: members.length}) }}
     </h2>
-    <div class="fr-table fr-table--layout-fixed">
-      <AdminTable :loading>
-        <thead>
-          <tr>
-            <AdminTableTh scope="col">{{ t("Members") }}</AdminTableTh>
-            <AdminTableTh scope="col">{{ t("Status") }}</AdminTableTh>
-            <AdminTableTh scope="col">{{ t("Member since") }}</AdminTableTh>
-            <AdminTableTh scope="col">{{ t("Last connection") }}</AdminTableTh>
-            <AdminTableTh scope="col" v-if="isAdmin">{{ t("Action") }}</AdminTableTh>
-          </tr>
-        </thead>
-        <tbody>
-          <tr v-for="member in members" :key="member.user.id">
-            <td>
-              <p class="fr-text--bold fr-m-0">{{ member.user.first_name }} {{ member.user.last_name }}</p>
-              <p class="fr-m-0 fr-text--xs text-mention-grey f-italic">
-                <Vicon :height="11" :width="11" name="ri-mail-line"/>
-                {{ member.user.email }}
-              </p>
-            </td>
-            <td>
-              <AdminBadge :type="getStatusType(member.role)">{{ getStatus(member.role) }}</AdminBadge>
-            </td>
-            <td>{{ formatDate(member.since) }}</td>
-            <td>
-              <span v-if="member.user.last_login_at">{{ formatFromNow(member.user.last_login_at) }}</span>
-              <span v-else>{{ t("No connection") }}</span>
-            </td>
-            <td v-if="isAdmin">
-              <AdminEditMemberButton
-                :member="member"
-                :oid="oid"
-                :roles="roles"
-                @member-updated="updateMembers"
+    <AdminTable class="fr-table--layout-fixed" :loading>
+      <thead>
+        <tr>
+          <AdminTableTh scope="col">{{ t("Members") }}</AdminTableTh>
+          <AdminTableTh scope="col">{{ t("Status") }}</AdminTableTh>
+          <AdminTableTh scope="col">{{ t("Member since") }}</AdminTableTh>
+          <AdminTableTh scope="col">{{ t("Last connection") }}</AdminTableTh>
+          <AdminTableTh scope="col" v-if="isAdmin">{{ t("Action") }}</AdminTableTh>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="member in members" :key="member.user.id">
+          <td>
+            <p class="fr-text--bold fr-m-0">{{ member.user.first_name }} {{ member.user.last_name }}</p>
+            <p class="fr-m-0 fr-text--xs text-mention-grey f-italic inline-flex items-center">
+              <Vicon
+                :height="12"
+                :width="12"
+                name="ri-mail-line"
+                class="fr-mr-1v"
               />
-            </td>
-          </tr>
-        </tbody>
-      </AdminTable>
-    </div>
+              <Tooltip>
+                <TextClamp :text="member.user.email" :auto-resize="true" :max-lines="1" />
```

Comment:
I think we can now have the member email first letter slightly cropped
![image](https://github.com/user-attachments/assets/7b22897d-06ad-42f3-b8cf-5ba4dc6da408)